### DISCUSSION TITLE
SSO Contracts: TimestampAsserter Address

### DISCUSSION THREAD

**Discussion by 4NNNN** [zkSync-Community-Hub/zksync-developers#854] [SSO] [answered]
The session implementation uses the [TimestampAsserter](https://github.com/matter-labs/era-contracts/pull/1057#event-15254277006) and to make this work it should have been whitelisted on a system level. Please share us the whitelisted address of this contract on different networks.

**Comment by uF4No** [comment, marked as answer]
The way it works is that `TimestampAsserter` itself is the one whitelisted on the system so other contracts just call its methods to do any validations.

Some addresses on different ZK chains are:
- Era Sepolia testnet: 0xa64EC71Ee812ac62923c85cf0796aA58573c4Cf3 [see in explorer](https://sepolia.explorer.zksync.io/address/0xa64EC71Ee812ac62923c85cf0796aA58573c4Cf3)
- Sophon testnet: 0x5ea4c1df68fd54ea9242bc6c405e7699ebbcb5f1 [see in explorer](https://explorer.testnet.sophon.xyz/address/0x5ea4c1df68fd54ea9242bc6c405e7699ebbcb5f1)
- Sophon mainnet: 0xC3ad88249239d2AB7314C659e8Ad7Fd06C466652 [see in explorer](https://explorer.sophon.xyz/address/0xC3ad88249239d2AB7314C659e8Ad7Fd06C466652)
- Abstract testnet: 0x27570660a298db7373EaA50c1a728DA93b5BC969 [see in explorer](https://explorer.testnet.abs.xyz/address/0x27570660a298db7373EaA50c1a728DA93b5BC969)
